docs/excel/excel-add-ins-comments.md

```diff
@@ -197,6 +197,105 @@ Excel.run(function (context) {
+
+```js
+Excel.run(function (context) => {
+    var comments = context.workbook.worksheets.getActiveWorksheet().comments;
+
+    // Register the onAdded comment event handler.
+    comments.onAdded.add(commentAdded);
+
+    return context.sync();
+});
+
+function commentAdded() {
+    Excel.run(function (context) => {
+        // Retrieve the added comment using the comment ID.
+        // Note: This method assumes only a single comment is added at a time. 
+        var addedComment = context.workbook.comments.getItem(event.commentDetails[0].commentId);
+
+        // Load the added comment's data.
+        addedComment.load(["content", "authorName"]);
+
+        return context.sync().then(function () {
+            // Print out the added comment's data.
+            console.log(`A comment was added. ID: ${event.commentDetails[0].commentId}. Comment content:${addedComment.content}. Comment author:${addedComment.authorName}`);
+            return context.sync();
+        });            
+    });
+}
+```
+
+### Comment change events 
+The `onChanged` comment event is triggered in the following scenarios: 
+
+- A comment's content is updated.
+- A comment thread is resolved.
+- A comment thread is reopened.
+- A reply is added to a comment thread.
+- A reply is updated in a comment thread.
+- A reply is deleted in a comment thread.
+
+The following sample shows how to register the `onChanged` event handler and then use this event handler to log the changed comment's metadata. This sample assumes only a single comment has been changed. 
+
+```js
+Excel.run(function (context) => {
+    var comments = context.workbook.worksheets.getActiveWorksheet().comments;
+
+    // Register the onChanged comment event handler.
+    comments.onChanged.add(commentChanged);
+
+    return context.sync();
+});    
+
+function commentChanged() {
+    Excel.run(function (context) => {
+        // Retrieve the changed comment using the comment ID.
+        // Note: This method assumes only a single comment is changed at a time. 
+        var changedComment = context.workbook.comments.getItem(event.commentDetails[0].commentId);
+
+        // Load the changed comment's data.
+        changedComment.load(["content", "authorName"]);
+
+        return context.sync().then(function () {
+            // Print out the changed comment's data.
+            console.log(`A comment was changed. ID: ${event.commentDetails[0].commentId}`. Updated comment content: ${changedComment.content}`. Comment author: ${changedComment.authorName}`);
+            return context.sync();
+        });
+    });
+}
+```
+
+### Comment deletion events
+The `onDeleted` event is triggered when a comment is deleted from the comment collection. Once a comment has been deleted, its metadata is no longer available. The `commentId` is still available after a comment is deleted.  
```

Comment:
The "still available" note might be a little misleading, since it's only provided in the EventArgs and not available elsewhere. Maybe we can describe the use case a bit more? Maybe something like "The `CommentDeletedEventArgs` provide the comment IDs, in case your add-in is also managing individual comments."?